internal/core/converter/tensorflow/option.go

```diff
@@ -138,6 +152,19 @@ func WithFetches(operationNames []string, outputIndexes []int) Option {
 	}
 }
 
+// WithWarmupInputs returns Option that sets warmupInputs.
+func WithWarmupInputs(warmupInputs ...string) Option {
+	return func(t *tensorflow) {
+		if warmupInputs != nil {
+			if t.warmupInputs != nil {
+				t.warmupInputs = append(t.warmupInputs, warmupInputs...)
```

Comment:
thinking functional options should not append inputs instead of replacing the value only.
@hlts2 what do you think?

Reply:
I understand your opinion.
but From a flexibility point of view, I would like to keep the current implementation.